server/routes/v1.routes/secure-routes.js

```diff
@@ -0,0 +1,20 @@
+//This code handles a GET request for profile. 
```

Comment:
the sintax for routing files is name.routes.js being name the name of the file. If you are using icons on your vscode you are going to notice that the icon is different because express recognize that sintax.